experimental/devices/mfrc522/mfrc522.go

```diff
@@ -323,7 +346,12 @@ func (r *Dev) Request() (int, error) {
 func (r *Dev) Wait() error {
 	irqChannel := make(chan bool)
 	go func() {
-		irqChannel <- r.irqPin.WaitForEdge(r.operationTimeout)
+		result := r.irqPin.WaitForEdge(r.operationTimeout)
+		if r.isStopped {
```

Comment:
Assuming it's called in a separate thread, this is a memory access without lock. Modification to isStopped should be guarded by `mu sync.Mutex`